bodhi/server/util.py

```diff
@@ -297,9 +298,23 @@ def hostname(context=None):
 
 
 def markup(context, text):
-    return markdown.markdown(text, safe_mode="replace",
-                             html_replacement_text="--RAW HTML NOT ALLOWED--",
-                             extensions=['markdown.extensions.fenced_code'])
+    """
+    Return HTML from a markdown string
+
+    Args:
+        context:
```

Comment:
Did not know how to describe `context` here. Any thoughts?

Do we even need to pass context to this method?